### PR TITLE
v2v: add a new case for suppressing vddk error msg

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.vddk_errors.esx_70.rhev.rhv_upload.it_vddk.vpx_uri: PASS (413.27 s)

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -817,6 +817,14 @@ def vm_check(status_error):
                         qa_url, os.path.join(
                             qemu_guest_agent_dir, rpm_name))
 
+        if 'vddk_error' in checkpoint:
+            fqdn_record = params_get(params, 'fqdn_record')
+            cmd = 'echo %s >> /etc/hosts' % fqdn_record
+            with open('/etc/hosts', 'r') as fd:
+                if fqdn_record not in fd.read():
+                    LOG.debug('Write %s to /etc/hosts', fqdn_record)
+                    process.run(cmd, shell=True)
```
Comment (chunfuwen) [inline v2v/tests/src/function_test_esx.py:826]: Can we just use python native file write operation, not use process.run?

Comment (xiaodwan) [reply v2v/tests/src/function_test_esx.py]: Done, thanks.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**